test/functional/forall/test-forall-rangesegment.hpp

```diff
@@ -0,0 +1,76 @@
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+// Copyright (c) 2016-20, Lawrence Livermore National Security, LLC
+// and RAJA project contributors. See the RAJA/COPYRIGHT file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+
+#ifndef __TEST_FORALL_RANGESEGMENT_HPP__
+#define __TEST_FORALL_RANGESEGMENT_HPP__
+
+#include "RAJA/RAJA.hpp"
+#include "camp/resource.hpp"
+
+#include "test-forall.hpp"
+
+using namespace camp::resources;
+using namespace RAJA;
+using namespace camp;
```

Comment:
@mdavis36 I think you've scoped everything in this file properly. Do we need these namespaces?

Reply:
we do not, these are left over from refactoring into separate .cpp files, thank you